lib/krane/kubernetes_resource/daemon_set.rb

```diff
@@ -58,8 +58,11 @@ def relevant_pods_ready?
       return true if rollout_data["desiredNumberScheduled"].to_i == rollout_data["numberReady"].to_i # all pods ready
       relevant_node_names = @nodes.map(&:name)
       considered_pods = @pods.select { |p| relevant_node_names.include?(p.node_name) }
-      @logger.debug("Considered #{considered_pods.size} pods out of #{@pods.size} for #{@nodes.size} nodes")
-      considered_pods.present? && considered_pods.all?(&:deploy_succeeded?)
+      @logger.debug("DaemonSet is reporting #{rollout_data['numberReady']} pods ready")
```

Comment:
Yeah, originally I didn't want to do that to avoid an overly-long log line, but since it's debug that's not really a big concern. Fixed to make a single statement